src/main/resources/templates/user/create.html

```diff
@@ -116,12 +116,13 @@
                                     <option value="+373">+373 (Moldova)</option>
                                     <!-- Add more country codes as needed -->
                                 </select>
-                                <input type="text" name="mobile" th:field="*{mobile}" class="form-control mobile-number"
-                                       placeholder="phone number" id="mobile"
-                                       pattern="\d{10}" title="Phone number must be 10 digits only" required />
+                                <input type="text" name="phone" th:field="*{phone}" class="form-control mobile-number"
```

Comment:
The field name is `mobile`, please take a look at the class `UserCommand`, please revert this change.